.github/workflows/security-scan.yaml

```diff
@@ -68,29 +76,36 @@ jobs:
       - run: sleep 8
 
       - name: Run Dastardly scan on demo website
-        run: docker run
-          --add-host host.docker.internal:host-gateway
-          -e DASTARDLY_TARGET_URL=http://host.docker.internal:3001
-          -e DASTARDLY_OUTPUT_FILE=/dastardly/demo.dastardly-report.xml
-          -u root
-          -v $(pwd):/dastardly public.ecr.aws/portswigger/dastardly:latest
-          dastardly
+        id: demo_dastardly_scan
+        run: |
+          set +e     
+          docker run \
+          --add-host host.docker.internal:host-gateway \
+          -e DASTARDLY_TARGET_URL=http://host.docker.internal:3001 \
+          -e DASTARDLY_OUTPUT_FILE=/dastardly/demo.dastardly-report.xml \
+          -u root \
+          -v $(pwd):/dastardly public.ecr.aws/portswigger/dastardly:latest \
+          dastardly \
+      # echo "DASTARDLY_DEMO_EXIT_CODE=$?" >> $GITHUB_OUTPUT
```

Comment:
need to uncomment and test. also L66

Reply:
this is not working throwing error

Reply:
https://github.com/Tonomy-Foundation/Tonomy-App-Websites/actions/runs/5944917951/job/16123080379